zhaquirks/tuya/ts0601_cover.py

```diff
@@ -562,3 +579,74 @@ class TuyaCloneCover0601(TuyaWindowCover):
             }
         }
     }
+
+
+def register_v2_quirks() -> None:
+    """Register v2 Quirks."""
+
+    # Tuya window cover device.
+    #
+    # This variant supports:
+    #     - multiple data points included in tuya set_data_response.
+    #     - non-inverted control inputs,
+    #     - battery percentage remaining
+    #
+    # Most/all the quirks above are based on TuyaManufacturerWindowCover that only decodes
+    # ONE attribute from the Tuya set_data_response packet. This quirk is based on
+    # TuyaNewManufClusterForWindowCover which can handle multiple updates in one zigby frame.
+    #
+    # "_TZE200_eevqq1uv", "TS0601" - Zemismart ZM25R3 roller blind motor
+    add_to_registry_v2("_TZE200_68nvbio9", "TS0601"
+    ).also_applies_to("_TZE200_eevqq1uv", "TS0601"
+    ).replaces(TuyaNewManufClusterForWindowCover
+    ).adds(TuyaNewWindowCoverControl
+    ).adds(TuyaPowerConfigurationCluster
+    ).command_button(
+        WINDOW_COVER_COMMAND_SMALL_STEP_NAME,
+        TuyaNewWindowCoverControl.cluster_id,
+        None,
+        {"direction": CoverCommandStepDirection.Open},
+        entity_type=EntityType.STANDARD,
+        translation_key="small_step_open",
+    ).command_button(
+        WINDOW_COVER_COMMAND_SMALL_STEP_NAME,
```

Comment:
These command definitions create HA entities with ids like button.[device name]_none_[int]. I.e. there doesn't seem to be any sensible naming applied to the entity. is that expected behaviour, or something wrong with these definitions?

It's not really a problem, but feels like it could be better. Could the zha component use the translation key, command name, or another parameter to set the entity id?